### PR TITLE
Basic Google Analytics Support

### PR DESCRIPTION
**Pull request by wlach**
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

**Comment by wlach** [comment]
This adds basic google analytics support to this repository (based on the code I previously added to the [schema dictionary](https://github.com/mozilla/schema-dictionary)) -- it will help us answer the question: how many unique users at Mozilla are using this service? It will also give us some insight into what pages are most popular.

/cc @rafrombrc @mreid-moz @acmiyaguchi (who I also gave access to the Google Analytics account backing this)

File: src/ga.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const googleAnalytics = (gaID) => {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', gaID, 'auto');
+  ga('send', 'pageview');
+};
```
Comment (Iinh) [inline src/ga.js:9]: This was hard to read. Just wondering why you don't want to prettier-fy this?

**Review by Iinh — APPROVED** [review]
Thank you for implementing this @wlach!

File: src/App.svelte [hunk 2]
```diff
@@ -75,6 +75,11 @@
   page("/apps/:app/pings/:ping", setComponent(PingDetail));
   page("/apps/:app/metrics/:metric", setComponent(MetricDetail));
   page("/apps/:app", setComponent(AppDetail));
+  page.exit("*", (ctx, next) => {
+    ga("set", "page", ctx.page.current);
+    ga("send", "pageview");
+    next();
+  });
```
[the remark this answers is not in the document]
Comment (wlach) [reply src/App.svelte:82]: Yup!

File: src/ga.js (hunk 1 above)
Comment (wlach) [reply src/ga.js:9]: This is basically just a pre-minified extract from Google, formatting the chunk wouldn't really improve it substantially and makes it a little less recognizable: that bit of code is actually very familiar to many people who have worked with google analytics previously.

**Pull request merged**